stable/redis/templates/networkpolicy.yaml

```diff
@@ -31,6 +31,13 @@ spec:
             release: "{{ .Release.Name }}"
             role: metrics
             app: {{ template "redis.name" . }}
+      {{- if ge .Values.cluster.slaveCount 1.0 }}
+      - podSelector:
+          matchLabels:
+            release: "{{ .Release.Name }}"
+            role: slave
+            app: {{ template "redis.name" . }}
+      {{- end }}
```

Comment:
You should close the previous `if` statement:
```diff
+     {{- end }}
      {{- if ge .Values.cluster.slaveCount 1.0 }}
      - podSelector:
          matchLabels:
            release: "{{ .Release.Name }}"
            role: slave
            app: {{ template "redis.name" . }}
-     {{- end }}
```

Reply:
@juan131 thanks for the catch. I updated the PR